packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -465,7 +469,6 @@ export interface CallCompositeProps extends BaseCompositeProps<CallCompositeIcon
     callInvitationUrl?: string;
     formFactor?: 'desktop' | 'mobile';
```

Comment:
Interesting question comes to my mind - should we move  form factor into options once we are allowed to make breaking change? @prprabhu-ms @JamesBurnside

Reply:
Yeah I agree we can move form factor into the CallComposite options object prop. But once we are allowed to make breaking changes and for another PR